bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "split-test",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
This isn't really a new feature release - should we just bump the patch instead of the minor?